### PR TITLE
Book info

### PR DESCRIPTION
**Pull request by prem2597**

### CODE REVIEW

File: application.py [hunk 1]
```diff
@@ -0,0 +1,143 @@
...
+        first_name = request.form.get("first-name")
+        last_name = request.form.get("last-name")
+        full_name = first_name + " " + last_name
+        birth = request.form.get("date-of-birth")
+        gen = request.form.get("gender")
+        mail = request.form.get("email")
+        ph = request.form.get("Phone-number")
+        pswd = request.form.get("password")
+        # details = full_name +"\n"+ birth +"\n"+ gen +"\n"+ mail + "\n" + ph
+        user1 = Users(name=full_name, dob=birth, gender=gen, email=mail, phone=ph, password=pswd, timestamp=datetime.now())
```
Comment (satya-rayi) [inline application.py]: are you encrypting password?

File: application.py [hunk 2]
```diff
@@ -0,0 +1,143 @@
...
+#     return render_template("search.html", booklist = bookdata)
+
+@app.route("/book/<isbn>", methods=["GET"])
+def bookInfo(isbn):
+    # pass
+    response = goodread_api(isbn)
+    return render_template("bookInfo.html", Name = response["name"], Author = response["author"], ISBN = response["isbn"], Year = response["year"], rating = response["average_rating"], count = response["reviews_count"], image = response["img"])
+
+def goodread_api(isbn):
+    res = requests.get("https://www.goodreads.com/book/review_counts.json", params={"key": "0gaifU0ED4eOcG7fDno6g", "isbns": isbn})
```
Comment (satya-rayi) [inline application.py]: better add key as const variable, can be referred anywhere

File: application.py (hunk 1 above)
Comment (prem2597) [reply application.py:57]: Done with encryption sir

Comment (prem2597) [reply application.py]: > are you encrypting password?

Done with encryption sir

File: application.py (hunk 2 above)
Comment (prem2597) [reply application.py:117]: > better add key as const variable, can be referred anywhere

add the key as constant value sir

Comment (prem2597) [reply application.py]: > better add key as const variable, can be referred anywhere

added the key as constant value sir

**Pull request merged**